app/views/application/_flash.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="alerts">
+<div id="alerts" class="alerts">
```

Comment:
I'm not a big fan of ids to be honest, even though it would work perfectly fine here. To keep it consistent I'd rather use a data-role attribute. So in this case that might be `data-role="alerts"`. That way data attributes == javascript, classes == css, ids == link anchors.